.devcontainer/config.json

```diff
@@ -92,6 +92,11 @@
     "lock_key": "retention_manager_lock",
     "interval": 60000
   },
+  "telemetry_manager": {
+    "enabled": true,
+    "lock_key": "telemetry_manager_lock",
+    "interval": 10000
```

Comment:
Shouldn't it be 5 minutes (300000) from your PR description ?

Reply:
I don't think we need to change this file here, did you have a reason to update it ? otherwise default.json should be enough.

Reply:
yes you're right, I set the manager period via a constant in telemetryManager

Reply:
fixed